src/Services/Implementations/ReverseGeocodes/OpenStreetMapReverseGeocodeServiceBase.cs

```diff
@@ -33,12 +40,21 @@ public async Task<IEnumerable<string>> Get(Coordinate coordinate, List<PropertyI
 		return addressPropertyValueList;
 	}
 
-	public async Task<OpenStreetMapResponse?> SerializeFullResponse(Coordinate coordinate)
+	public async Task<OpenStreetMapResponse?> SerializeFullResponse(ReverseGeocodeRequest request)
 	{
 		try
 		{
-			var requestUri = RequestUri(coordinate);
+			var requestUri = RequestUri(request.Coordinate.Latitude, request.Coordinate.Longitude);
+
```

Comment:
There is no written rule yet, but to comply with previous usages. Is it ok for you to omit braces & remove empty lines ?
For empty lines, I think it's better to separate logical groups if there are many statements. For shorter and related codes these empty lines will look useless.